src/components/ScaleLattice.vue

```diff
@@ -0,0 +1,340 @@
+otonalFundamental
+<script setup lang="ts">
+import type { Interval, Scale } from "scale-workshop-core";
+import {
+  computed,
+  onMounted,
+  type ComputedRef,
+  ref,
+  onBeforeUnmount,
+} from "vue";
+import { PRIMES } from "xen-dev-utils";
+import { monzoEuclideanDistance, getPairCombinations } from "@/utils";
+const props = defineProps<{
+  scale: Scale;
+  width: number;
+  height: number;
+  heldScaleDegrees: Set<number>;
+}>();
+
+type Coord = { x: number; y: number };
+type Node = {
+  x: number;
+  y: number;
+  label: string;
+  index: number;
+  fill: string;
+};
+
+type Coords = {
+  [key: string]: Coord;
+};
+
+const coords: Coords = {
+  "1": {
+    x: 0,
+    y: 0,
+  },
+  "2": {
+    x: 0,
+    y: 0,
+  },
+  "3": {
+    x: 40,
+    y: 0,
+  },
+  "5": {
+    x: 0,
+    y: -40,
+  },
+  "7": {
+    x: 13,
+    y: -11,
+  },
+  "11": {
+    x: -14,
+    y: -18,
+  },
+  "13": {
+    x: -8,
+    y: -4,
+  },
+  "17": {
+    x: -5,
+    y: -32,
+  },
+  "19": {
+    x: 7,
+    y: -25,
+  },
+  "23": {
+    x: 20,
+    y: -6,
+  },
+};
+
+const container = ref<HTMLDivElement | null>(null);
+
+const adjustLatticeSize = () => {
```

Comment:
I'm inclined to believe that these acrobatics can be avoided by setting the `<svg>`'s `width` and `height` attributes to `100%` and carefully setting the `viewBox` and `preserveAspectRatio` attributes with respect to the coords or vice versa.
Not a blocker though. Can be deferred to a separate issue to be solved later.

Reply:
Yeah, I am not an svg expert so if you know how to do it I would appreciate some more concrete suggestions.

Reply:
Let's defer to a later issue. I need to sit down to solve it so I cannot give a concrete solution while just glancing over.